modules/logging/aws-for-fluent-bit/main.tf

```diff
@@ -0,0 +1,116 @@
+      type        = "Federated"
+    }
+  }
+}
+
+resource "aws_iam_role" "aws_for_fluent_bit_iam_role" {
+  name               = "${var.cluster_name}-aws-for-fluent-bit"
+  assume_role_policy = data.aws_iam_policy_document.aws_for_fluent_bit_assume_role_policy.json
+
+  tags = {
+    Environment = var.environment
+    Project     = var.project
+  }
+}
+
+// Configuration: https://github.com/aws/eks-charts/blob/master/stable/aws-for-fluent-bit/README.md
+resource "helm_release" "aws-for-fluent-bit" {
+  depends_on = [
+    var.module_depends_on
+  ]
+
+  name       = "aws-for-fluent-bit"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-for-fluent-bit"
+  version    = "0.1.3"
+  namespace  = local.namespace
+
+  values = [
+    templatefile("${path.module}/values/aws-for-fluent-bit.yaml", {
+      aws_region = var.aws_region
+      namespace  = local.namespace
+
+      service_account_auto_create              = var.service_account_auto_create
+      service_account_name                     = var.service_account_name
+      service_account_role_arn                 = aws_iam_role.aws_for_fluent_bit_iam_role.arn
+
+      daemon_set_resources_limits_cpu          = var.daemon_set_resources_limits_cpu
+      daemon_set_resources_limits_memory       = var.daemon_set_resources_limits_memory
+      daemon_set_resources_requests_cpu        = var.daemon_set_resources_requests_cpu
+      daemon_set_resources_requests_memory     = var.daemon_set_resources_requests_memory
+
+      cloudwatch_enabled                       = var.cloudwatch_enabled
+      cloudwatch_match                         = var.cloudwatch_match
+      cloudwatch_log_group_name                = var.cloudwatch_log_group_name
+      cloudwatch_log_stream_name               = var.cloudwatch_log_stream_name
+      cloudwatch_log_stream_prefix             = var.cloudwatch_log_stream_prefix
+      cloudwatch_log_key                       = var.cloudwatch_log_key
+      cloudwatch_log_format                    = var.cloudwatch_log_format
+      cloudwatch_auto_create_group             = var.cloudwatch_auto_create_group
+      cloudwatch_endpoint                      = var.cloudwatch_endpoint
+      cloudwatch_credentials_endpoint          = var.cloudwatch_credentials_endpoint
+      cloudwatch_cross_account_role_arn        = var.cloudwatch_cross_account_role_arn
+
+      firehose_enabled                         = var.firehose_enabled
+      firehose_match                           = var.firehose_match
+      firehose_delivery_stream                 = var.firehose_delivery_stream
+      firehose_data_keys                       = var.firehose_data_keys
+      firehose_endpoint                        = var.firehose_endpoint
+      firehose_time_key                        = var.firehose_time_key
+      firehose_time_key_format                 = var.firehose_time_key_format
+      firehose_cross_account_role_arn          = var.firehose_cross_account_role_arn
```

Comment:
This params and variables still needs?

Reply:
https://github.com/provectus/swiss-army-kube/pull/105#discussion_r523717989